README.md

```diff
@@ -1 +1,37 @@
-# charts
\ No newline at end of file
+<div align="center">
+
+# SUSE Edge Helm Charts
+
+<p align="center">
+  <img alt="SUSE Logo" src="https://www.suse.com/assets/img/suse-white-logo-green.svg" height="140" />
+  <h3 align="center">SUSE Edge misc</h3>
+</p>
+
+| :warning: **This is an unofficial and unsupported repository. See the [official documentation](https://www.suse.com/solutions/edge-computing/).** |
+| --- |
+
+</div>
+
+# SUSE Edge Helm Charts
+
+A repository that contains different Helm Charts from the SUSE Edge engineering team
+
+> :warning: This is an unofficial tool, don't blame us!
+
+## Helm Charts
+| Chart | Description |
+| - | - |
+| [Akri](charts/akri) | A Kubernetes Resource Interface for the Edge |
+
+# Generate index.html
+
+```
+helm plugin install https://github.com/halkeye/helm-repo-html
+# Generate your yaml and html
+helm repo index ./
+helm repo-html -t index.tpl
+# Now save the two file
+git add index.yaml index.html
```

Comment:
Same here the `index.yaml` should already be handled when generating the `index.html`